src/poetry_plugin_bundle/console/commands/bundle/venv.py

```diff
@@ -41,8 +47,21 @@ class BundleVenvCommand(BundleCommand):
 
     bundler_name = "venv"
 
+    @property
+    def activated_extras(self) -> set[str]:
+        extras: set[str] = set()
+        # NOTE: simplified version of InstallCommand option handling
+        if self.option("all-extras"):
+            extras.update(self.poetry.package.extras)
+        for extra in self.option("extras", []):
+            extras.update(extra.split())
```

Comment:
`poetry install` does not accept both flags at the same time. Maybe, we should print the same error for consistency?